Descriptors/PerformanceLevelDescriptor.xml

```diff
@@ -19,63 +19,69 @@
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Fail</CodeValue>
-		<ShortDescription>Fail</ShortDescription>
-		<Description>Fail</Description>
+		<CodeValue>Well Below Basic</CodeValue>
+		<ShortDescription>Well Below Basic</ShortDescription>
+		<Description>Well Below Basic</Description>
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Pass</CodeValue>
-		<ShortDescription>Pass</ShortDescription>
-		<Description>Pass</Description>
+		<CodeValue>Met Standard</CodeValue>
+		<ShortDescription>Met Standard</ShortDescription>
+		<Description>Met Standard</Description>
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Proficient</CodeValue>
-		<ShortDescription>Proficient</ShortDescription>
-		<Description>Proficient</Description>
+		<CodeValue>Did Not Meet Standard</CodeValue>
+		<ShortDescription>Did Not Meet Standard</ShortDescription>
+		<Description>Did Not Meet Standard</Description>
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Well Below Basic</CodeValue>
-		<ShortDescription>Well Below Basic</ShortDescription>
-		<Description>Well Below Basic</Description>
+		<CodeValue>Pass</CodeValue>
+		<ShortDescription>DEPRECATE: Pass</ShortDescription>
```

Comment:
Ticket comment mentions "Keep Pass/Fail" but this is deprecating them, did the final decision on this change?

Reply:
@ejansson, did we adjust the requirements here or should Pass/Fail be included? I don't see any notes in the ticket so I may have just been overzealous in deprecating those values.

Reply:
Going ahead and approving this, but we should either adjust the ticket or this change to match.

Reply:
removed the pass/fail deprecated designation to align with the ticket.

@blmeyers, @ejansson